README.md

```diff
@@ -166,11 +166,11 @@ We'd greatly appreciate if you could:
 
 | **Name** | **Description**| **Deal details** | **Validity** | **Latest** |
 | :- | :- | :- | :- | :- | 
-| [Practical UI](https://www.practical-ui.com/design-system/) | Lean and powerful Figma design system and UI kit | **BLACKFRIDAY24** - 40% off | Ends Dec 2 |
+| [Practical UI](https://www.practical-ui.com/design-system/) | Lean and powerful Figma design system and UI kit | **PDWBGHXR30** - 30% off | - | ✅ | 
```

Comment:
This is a purchasing power parity discount only available for you, it's not a black friday discount